react/features/virtual-background/components/VirtualBackgroundDialog.js

```diff
@@ -0,0 +1,128 @@
+// @flow
+/* eslint-disable react/jsx-no-bind, no-return-assign */
+import React, { useState } from 'react';
+
+import { Dialog } from '../../base/dialog';
+import { translate } from '../../base/i18n';
+import { Icon, IconBlurBackground } from '../../base/icons';
+import { connect } from '../../base/redux';
+import { Tooltip } from '../../base/tooltip';
+import { toggleBackgroundEffect } from '../../blur/actions';
```

Comment:
please rename features/blur to features/virtual-backgrounds

Reply:
Line 144 -> onError = { event => event.target.style.display = 'none' }